routing/blinding_test.go

```diff
@@ -53,3 +57,106 @@ func TestBlindedPathValidation(t *testing.T) {
 		})
 	}
 }
+
+// TestBlindedPaymentToHints tests conversion of a blinded path to a chain of
+// route hints. As our function assumes that the blinded payment has already
+// been validated (ie, has at least 2 hops), we don't test cases with an invalid
+// number of hops.
+func TestBlindedPaymentToHints(t *testing.T) {
+	t.Parallel()
+
+	var (
+		_, pk1  = btcec.PrivKeyFromBytes([]byte{1})
+		_, pkb1 = btcec.PrivKeyFromBytes([]byte{2})
```

Comment:
Would they? `pk1` is the introduction node and `pkb1` is the [blinded introduction node](https://github.com/lightningnetwork/lightning-onion/pull/57/files#diff-6883260b5ee8afaff7c13850ceede938210b4390bdf049068419912202c214beR89)  - the value will be different even though we don't use it. 

I specifically wanted different values so that a test would fall over if we're using the wrong thing.